tests/integration/test_ipython.py

```diff
@@ -137,3 +137,35 @@ def test_passing_valid_argument(self, tmpdir, capsys):
         assert "<iframe" in html
         assert "flamegraph.html" in html
         assert "Results saved to" in stdout
+
+    @pytest.mark.parametrize(
+        "args,title",
+        [
+            ("--temporary-allocation-threshold=2", "flamegraph report"),
+            ("--leaks", "flamegraph report (memory leaks)"),
+            ("--temporal", "temporal flamegraph report"),
+            ("--leaks --temporal", "temporal flamegraph report (memory leaks)"),
+        ],
+    )
+    def test_report_title_by_report_type(self, tmpdir, capsys, args, title):
```

Comment:
Maybe is worth checking the error when --temporal and `temporary_allocation_threshold>=0`

Reply:
Ah, good call, can do.